assemble/bin/tool.sh

```diff
@@ -36,7 +36,10 @@ if [[ -z "$ZOOKEEPER_HOME" ]] ; then
    exit 1
 fi
 
-ZOOKEEPER_CMD='ls -1 $ZOOKEEPER_HOME/zookeeper-[0-9]*[^csn].jar '
+ZOOKEEPER_CMD='ls -1 $ZOOKEEPER_HOME/lib/zookeeper-[0-9]*[^csn].jar '
+if [[ "${ZOOKEEPER_VERSION}" = 3.[01234].* ]]; then
```

Comment:
I find the regex pattern matching feature of bash to be more reliable than the glob equality feature. Also, the quotes and braces on the left side aren't neeeded.

However, a more fundamental problem: where is `ZOOKEEPER_VERSION` defined? There is no guarantee it is defined anywhere by anything that calls this script. It should probably do something like what `assemble/bin/start-all.sh` does.

```suggestion
ZOOKEEPER_VERSION=$(find -L "$ZOOKEEPER_HOME" -maxdepth 2 -name "zookeeper-[0-9]*.jar" | head -1)
if [[ $ZOOKEEPER_VERSION =~ ^3[.][01234].*$ ]]; then
```